rust/bridge/node/futures/src/executor.rs

```diff
@@ -43,29 +39,44 @@ impl<T: Future> Future for AssertSendSafe<T> {
     }
 }
 
-/// Adds support for executing closures and futures on the JavaScript main thread's microtask queue.
+/// Adds support for executing closures and futures on the JavaScript main thread's event queue.
```

Comment:
This was my mistake; we're using *Node's* [event queue](https://nodejs.org/api/n-api.html#n_api_asynchronous_thread_safe_function_calls) rather than *JavaScript's* [microtask queue](https://developer.mozilla.org/en-US/docs/Web/API/HTML_DOM_API/Microtask_guide).